src/software/networking/nanopb_primitive_set_multicast_listener.cpp

```diff
@@ -0,0 +1,36 @@
+#include "software/networking/nanopb_primitive_set_multicast_listener.h"
+
+#include "pb_decode.h"
+
+extern "C"
+{
+#include "shared/proto/tbots_software_msgs.nanopb.h"
```

Comment:
nit: this include is duplicated in the header

Reply:
good catch, removed